tests/template_tests/test_nodelist.py

```diff
@@ -32,6 +33,21 @@ def test_ifchanged(self):
         self.assertEqual(len(vars), 1)
 
 
+class TextNodeReprTest(TestCase):
```

Comment:
I'd combine this with the test class above or at least use a more generic name like `TextNodeTests` so more tests can be added without changing the class name.